### DISCUSSION TITLE
Slim Application Error

### DISCUSSION THREAD

**Discussion by wachirawamwangi** [nextcloud/all-in-one#2895] [Questions] [answered]
I am having issues  running nextcloud on my digitalocean droplet and I get this error.  

![Screenshot from 2023-07-01 19-51-52](https://github.com/nextcloud/all-in-one/assets/61097005/21c53776-e9eb-444e-84a4-dade3ccbb590)

I have seen similar issues raised here but the only comments there advise posters to create a new issues.
OS is Ubuntu 20.04
I  have several other services running meaning ports 80, 8080 and 443 are already occupied.
I have a domain with two subdomains, with one of the subdomains - with ssl - already running in the same droplet.
I am however able to access the setup page but once I enter the password in the login page it brings this error.
I am new to everything docker, reverse proxy etc etc
Thank you in advance for your assistance

**Comment by cedrikletarte** [comment]
I have the same problem in this discussion : https://github.com/nextcloud/all-in-one/discussions/2898

**Comment by szaimen** [comment, marked as answer]
Hi, it looks like port 443 is already used on your server. Please follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md thus.
